runner/docker/Dockerfile.live-base-comfyui-sam2

```
@@ -18,6 +18,9 @@ RUN pip install --no-cache-dir --upgrade pip==${PIP_VERSION} setuptools==69.5.1
 # Install Sam2 dependencies
 RUN pip install --no-cache-dir torch==2.3.1 torchvision torchaudio
 
+# Set TORCH_CUDA_ARCH_LIST environment variable, fixes build error in segment-anything-2-real-time
+ENV TORCH_CUDA_ARCH_LIST="6.0 7.0 7.5 8.0 8.6+PTX"
```

I believe we can probably set this to "8.0 8.6+PTX 8.7 9.0 9.0a" since only ada and hopper should be supported for these workflows ( I think >= ampere is not capable of realtime with these models )


nvcc tag | TORCH_CUDA_ARCH_LIST | GPU Arch | Year | eg. GPU
-- | -- | -- | -- | --
sm_50, sm_52 and sm_53 | 5.0 5.1 5.3 | Maxwell support | 2014 | GTX 9xx
sm_60, sm_61, and sm_62 | 6.0 6.1 6.2 | Pascal support | 2016 | 10xx, Pxxx
sm_70 and sm_72 | 7.0 7.2 | Volta support | 2017 | Titan V
sm_75 | 7.5 | Turing support | 2018 | most 20xx
sm_80, sm_86 and sm_87 | 8.0 8.6 8.7 | Ampere support | 2020 | RTX 30xx, Axx[xx]
sm_89 | 8.9 | Ada support | 2022 | RTX xxxx 40xx L4xx
sm_90, sm_90a | 9.0 9.0a | Hopper support | 2022 | H100

nvcc tag	TORCH_CUDA_ARCH_LIST	GPU Arch	Year	eg. GPU
sm_50, sm_52 and sm_53	5.0 5.1 5.3	[Maxwell](https://en.wikipedia.org/wiki/Maxwell_(microarchitecture)) support	2014	GTX 9xx
sm_60, sm_61, and sm_62	6.0 6.1 6.2	[Pascal](https://en.wikipedia.org/wiki/Pascal_(microarchitecture)) support	2016	10xx, Pxxx
sm_70 and sm_72	7.0 7.2	[Volta](https://en.wikipedia.org/wiki/Volta_(microarchitecture)) support	2017	Titan V
sm_75	7.5	[Turing](https://en.wikipedia.org/wiki/Turing_(microarchitecture)) support	2018	most 20xx
sm_80, sm_86 and sm_87	8.0 8.6 8.7	[Ampere](https://en.wikipedia.org/wiki/Ampere_(microarchitecture)) support	2020	RTX 30xx, Axx[xx]
sm_89	8.9	[Ada](https://en.wikipedia.org/wiki/Ada_Lovelace_(microarchitecture)) support	2022	RTX xxxx 40xx L4xx
sm_90, sm_90a	9.0 9.0a	[Hopper](https://en.wikipedia.org/wiki/Hopper_(microarchitecture)) support	2022	H100

https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list

---

Nice research! The + should include any newer architecture, so the following should work:
```suggestion
ENV TORCH_CUDA_ARCH_LIST="8.0+PTX"
```